dotnet/src/SemanticKernel.Abstractions/AI/FunctionChoiceBehaviors/FunctionChoiceBehavior.cs

```diff
@@ -30,6 +31,32 @@ public static FunctionChoiceBehavior Auto(IEnumerable<KernelFunction>? functions
         return new AutoFunctionChoiceBehavior(functions, autoInvoke);
     }
 
+    /// <summary>
+    /// Gets an instance of the <see cref="FunctionChoiceBehavior"/> that provides either all of the <see cref="Kernel"/>'s plugins' functions to the AI model to call or specific ones.
+    /// This behavior forces the model to always call one or more functions. The model will then select which function(s) to call.
```

Comment:
This line seems somewhat contradictory.  On the one hand we are saying we force the model to call one or more functions, but then we say the model chooses which ones to call. So when we pass 3 functions to the model, does it have to call all three, or does it choose from the three?  Might be worth rewording to clarify.

Reply:
Removed the confusing sentence.